chart/operator/templates/grafana-k8sgpt-dashboard.yaml

```diff
@@ -15,4 +15,25 @@ metadata:
 data:
   k8sgpt-overview.json: |
 {{ .Files.Get "dashboards/k8sgpt-overview.json" | indent 4}}
+---
+{{ if .Values.grafanaDashboard.grafanaOperator.enabled -}}
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  labels:
+    app.kubernetes.io/component: grafana-dashboard
+    app.kubernetes.io/created-by: k8sgpt-operator
+    app.kubernetes.io/part-of: k8sgpt-operator
+  {{- include "chart.labels" . | nindent 4 }}
+  name: k8sgpt-overview
+  namespace: {{ .Values.grafanaDashboard.namespace | default (include "k8sgpt-operator.namespace" .) }}
+spec:
+  folder: {{ .Values.grafanaDashboard.folder.name }}
```

Comment:
.Values.grafanaDashboard.folder.name does not exist, shall we placeholder?

Reply:
Hm I actually just reused it from https://github.com/k8sgpt-ai/k8sgpt-operator/blob/a5a0283f6239787c5264e388b3347001c6a93401/chart/operator/templates/grafana-k8sgpt-dashboard.yaml#L6

and it is defined in https://github.com/k8sgpt-ai/k8sgpt-operator/blob/0a32492cc1b3b7d7211169f40652b09feab5ff84/chart/operator/values.yaml#L12

Reply:
I presume this is due to the prometheus operator having grafana as a sub chart

Reply:
I am afraid I don't understand the discussion. Maybe you can help me with that here ... I can only say that https://github.com/k8sgpt-ai/k8sgpt-operator/blob/main/chart/operator/values.yaml already defines .Values.grafanaDashboard and that is what I use in https://github.com/k8sgpt-ai/k8sgpt-operator/blob/a5a0283f6239787c5264e388b3347001c6a93401/chart/operator/templates/grafana-k8sgpt-dashboard.yaml .. However, maybe I overlook something you think of. Sorry that I don't see the problem here ...